File: Java/Produk.java
public class Produk {
    private String idProduk;
    private String nama;
    private int harga;

    public Produk() {}

    public Produk(String idProduk, String nama, int harga) {
        this.idProduk = idProduk;
        this.nama = nama;
        this.harga = harga;
    }

    public String getIdProduk() { return idProduk; }
    public void setIdProduk(String idProduk) { this.idProduk = idProduk; }

    public String getNama() { return nama; }
    public void setNama(String nama) { this.nama = nama; }

    public int getHarga() { return harga; }
    public void setHarga(int harga) { this.harga = harga; }
}

File: Python/Produk.py
class Produk:
    def __init__(self, id_produk="", nama="", harga=0):
        self.__id_produk = id_produk
        self.__nama = nama
        self.__harga = harga

    # Getter & Setter
    def get_id_produk(self): return self.__id_produk
    def set_id_produk(self, id_produk): self.__id_produk = id_produk

    def get_nama(self): return self.__nama
    def set_nama(self, nama): self.__nama = nama

    def get_harga(self): return self.__harga
    def set_harga(self, harga): self.__harga = harga
